src/import-data/ImportData.ts

```diff
@@ -65,16 +65,29 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
   const importPFB = async (importRequest: PFBImportRequest, workspace: WorkspaceInfo) => {
     if (workspace.cloudPlatform === 'Azure') {
       const wdsUrl = await Ajax().Apps.listAppsV2(workspace.workspaceId).then(resolveWdsUrl);
-      await Ajax().WorkspaceData.startImportJob(wdsUrl, workspace.workspaceId, {
+      const { namespace, name, cloudPlatform } = workspace;
+      const { jobId } = await Ajax().WorkspaceData.startImportJob(wdsUrl, workspace.workspaceId, {
         url: importRequest.url.toString(),
         type: 'PFB',
       });
+      // TODO a better way to make this conform to expected shape?
```

Comment:
+1 on this TODO but not sure the best way...

Reply:
Maybe an intermediate variable with a type declaration?

```ts
const newJob: ImportJob = { ... }
```

I also might suggest getting rid of the FP trappings here and replacing Utils.append with an inline function to make things clearer.

```ts
asyncImportJobStore.update((previousJobs) => [...previousJobs, newJob])
```